content/pt-br/docs/concepts/cluster-administration/manage-deployment.md

```diff
@@ -0,0 +1,462 @@
+(imprimindo cada recurso como _resource/name_). Em seguida `grep` apenas o "serviço", e então o imprimimos com `kubectl get`.
+
+Se você organizar seus recursos em vários subdiretórios dentro de um diretório específico, você também pode executar recursivamente as operações nos subdiretórios, especificando
+com `--recursive` ou `-R` junto com a _flag_ `--filename,-f`.
+
+Por exemplo, suponha que exista um diretório `project/k8s/development` que contém todos os {{< glossary_tooltip text="manifests" term_id="manifest" >}} necessários para o ambiente de desenvolvimento, organizados por tipo de recurso:
+
+```
+project/k8s/development
+├── configmap
+│   └── my-configmap.yaml
+├── deployment
+│   └── my-deployment.yaml
+└── pvc
+    └── my-pvc.yaml
+```
+
+Por padrão, executar uma operação em massa em `project/k8s/development` irá parar no primeiro nível do diretório, não processando nenhum subdiretório. Se tivéssemos tentado criar os recursos neste diretório usando o seguinte comando, teríamos encrotado um erro:
+
+```shell
+kubectl apply -f project/k8s/development
+```
+
+```shell
+error: you must provide one or more resources by argument or filename (.json|.yaml|.yml|stdin)
+```
+
+Em invés disso, utilize a _flag_ `--recursive` ou `-R` com a _flag_ `--filename,-f` assim:
+
+```shell
+kubectl apply -f project/k8s/development --recursive
+```
+
+```shell
+configmap/my-config created
+deployment.apps/my-deployment created
+persistentvolumeclaim/my-pvc created
+```
+
+A _flag_ `--recursive` funciona com qualquer operação que aceite a _flag_ `--filename,-f` como: `kubectl {create,get,delete,describe,rollout}` etc.
+
+A _flag_ `--recursive` também funciona quando múltiplos `-f` são usados como argumento:
+
+```shell
+kubectl apply -f project/k8s/namespaces -f project/k8s/development --recursive
+```
+
+```shell
+namespace/development created
+namespace/staging created
+configmap/my-config created
+deployment.apps/my-deployment created
+persistentvolumeclaim/my-pvc created
+```
+
+Se você está interessado em aprender mais sobre `kubectl`, vá em frente e leia [kubectl _Overview_](/docs/reference/kubectl/overview/).
+
+## Usando _labels_ efetivamente
+
+
+Os exemplos que usamos até agora aplicam no máximo uma única _label_ a qualquer recurso. Há muitos cenários em que múltiplas _labels_ devem ser usadas ​​para distinguir conjuntos uns dos outros.
+
+Por exemplo, diferentes aplicações podem usar diferentes valores para a _label_ `app`, mas uma aplicação multi-nível, como [_guestbook example_](https://github.com/kubernetes/examples/tree/master/guestbook/), também precisaria distinguir cada _tier_. O _frontend_ pode carregar as seguintes _labels_:
+
+```yaml
+     labels:
+        app: guestbook
+        tier: frontend
+```
+
+Enquanto o Redis _master_ e _slave_ teriam _labels_ de diferentes `tier`, e talvez até uma label adicional para `role`:
```

Comment:
Maybe you can change the words `master` and `slave` to others synonymous, we aren't not using these words anymore at the docs.

Reply:
i changed to "primário" and "secundário" using this [link](https://en.wikipedia.org/wiki/Master/slave_(technology)#Possible_replacement_conventions) as a reference